### PR TITLE
Add Ren'Py 7.3.2

### PR DESCRIPTION
**Pull request by MunsterPlop**

### CODE REVIEW

File: bucket/renpy.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+    "homepage": "https://www.renpy.org/",
+    "description": "Popular open source visual novel engine that uses Python for scripting",
+    "license": "MIT",
+    "version": "7.3.2",
+    "url": "https://www.renpy.org/dl/7.3.2/renpy-7.3.2-sdk.7z.exe#dl.7z",
```
Comment (Ash258) [inline bucket/renpy.json]:
```suggestion
    "url": "https://www.renpy.org/dl/7.3.2/renpy-7.3.2-sdk.7z.exe#/dl.7z",
```

File: bucket/renpy.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "bin": "renpy.exe",
+    "shortcuts": [
+        [
+            "renpy.exe",
+            "RenPy"
+        ]
+    ],
+    "checkver": {
+        "url": "https://www.renpy.org/release_list.html",
+        "regex": "Ren'Py ([\\d\\.]+)"
```
Comment (Ash258) [inline bucket/renpy.json]:
```suggestion
        "regex": "Ren'Py ([\\d.]+)"
```

File: bucket/renpy.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    },
+    "autoupdate": {
+        "url": "https://www.renpy.org/dl/$version/renpy-$version-sdk.7z.exe#dl.7z",
```
Comment (Ash258) [inline bucket/renpy.json]:
```suggestion
        "url": "https://www.renpy.org/dl/$version/renpy-$version-sdk.7z.exe#/dl.7z",
```

File: bucket/renpy.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+        "hash": {
+            "url": "https://www.renpy.org/dl/$version/checksums.txt",
```
Comment (Ash258) [inline bucket/renpy.json]:
```suggestion
            "url": "$baseurl/checksums.txt",
```

File: bucket/renpy.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+            "regex": "$sha256.*?$basename"
```
Comment (Ash258) [inline bucket/renpy.json]: I think this could be removed. But needs to be tested.

```suggestion
```

Comment (MunsterPlop) [reply bucket/renpy.json]: I just tested it and it works fine but returns the md5 checksum by default. 
If that's okay with you, I'm commiting this.

Comment (Ash258) [reply bucket/renpy.json]: Ok then go with this. No need to use greedy match all when it is only whitespace.

```suggestion
            "regex": "$sha256\\s+$basename"
```

**Comment by r15ch13** [comment]
Even if it's a tool to create games, I would suggest adding it to the games bucket.
https://github.com/Calinou/scoop-games

**Comment by MunsterPlop** [comment]
> 
> 
> Even if it's a tool to create games, I would suggest adding it to the games bucket.
> https://github.com/Calinou/scoop-games

I got nothing against that but some game engines such as Godot are already in the extras bucket, so I'd ask @Ash258 for some input on this.

**Comment by Ash258** [comment]
I do not mind it here. Since cheat engine is here for example too.

**Pull request merged**